ibis/backends/pyspark/compiler.py

```diff
@@ -70,6 +70,7 @@ class PySparkCompiler(SQLGlotCompiler):
         ops.Hash: "hash",
         ops.Log10: "log10",
         ops.LStrip: "ltrim",
+        ops.RandomUUID: "uuid",
```

Comment:
@jcrist  There was a bug in the previous version of Spark, it has been addressed in the latest 4.0 version.(spark jira [issue](https://issues.apache.org/jira/browse/SPARK-47104) and [pr](https://github.com/apache/spark/pull/45199)).

Reply:
We don't currently support spark 4.0, so I don't think we should provide this mapping.

Reply:
Could we open an issue documenting this so when we support spark 4.0 (not sure how realistic that is) then this could be a nice onboarding issue?

Reply:
I will remove pyspark uuid support for now and open an issue for future reference.